ddtrace/logger.py

```diff
@@ -0,0 +1,30 @@
+import logging
+from logging.handlers import RotatingFileHandler
+
+from ddtrace.internal.utils.formats import asbool
+from ddtrace.utils.formats import get_env
+
+
+def configure_logger():
+    logging_level = asbool(get_env("trace", "debug", default=False))
+    log_path = get_env("trace", "log", "file", default="default")
```

Comment:
You could have the default be `None` here instead. then you can do `if log_path is not None` below.

Reply:
As an update, this has been updated with `debug_log_level = os.environ.get("DD_TRACE_DEBUG", "false").lower() in ("true", "1")`, so I'll mark this as resolved and can reopen if this can be further adjusted!